spec.emu

```diff
@@ -699,27 +687,25 @@ location: https://github.com/tc39/proposal-decimal/
       1. If _d1_ or _d2_ is *NaN*<sub>𝔻</sub>, return Decimal128ValueToObject(*NaN*<sub>𝔻</sub>).
       1. If _d1_ or _d2_ is *+∞*<sub>𝔻</sub>, then
         1. If _d1_ or _d2_ is a zero Decimal128 value, return Decimal128ValueToObject(*NaN*<sub>𝔻</sub>).
-        1. If _d1_ or _d2_ is negative, return Decimal128ValueToObject(*-∞*<sub>𝔻</sub>).
+        1. If sign(_d1_) = -1 or sign(_d2_) = -1, return Decimal128ValueToObject(*-∞*<sub>𝔻</sub>).
         1. Return Decimal128ValueToObject(*+∞*<sub>𝔻</sub>).
       1. If _d1_ or _d2_ is *-∞*<sub>𝔻</sub>, then
         1. If _d1_ or _d2_ is a zero Decimal128 value, return Decimal128ValueToObject(*NaN*<sub>𝔻</sub>).
-        1. If _d1_ and _d2_ are both negative, return Decimal128ValueToObject(*+∞*<sub>𝔻</sub>).
+        1. If sign(_d1_) = -1 and sign(_d2_) = -1, return Decimal128ValueToObject(*+∞*<sub>𝔻</sub>).
         1. Return Decimal128ValueToObject(*-∞*<sub>𝔻</sub>).
       1. Let _v1_ be cohort(_d1_).
       1. Let _v2_ be cohort(_d1_).
       1. Let _q1_ be quantum(_d1_).
       1. Let _q2_ be quantum(_d2_).
       1. Let _newQ_ be _q1_ + _q2_.
       1. If _v1_ or _v2_ is *+0*<sub>𝔻</sub>, then
-        1. If _d1_ or _d2_ is negative, return Decimal128ValueToObject(PickQuantum(*-0*<sub>𝔻</sub>, _newQ_)).
+        1. If sign(_d1_) = -1 or sign(_d2_) = -1, return Decimal128ValueToObject(PickQuantum(*-0*<sub>𝔻</sub>, _newQ_)).
         1. Return Decimal128ValueToObject(PickQuantum(*+0*<sub>𝔻</sub>, _newQ_)).
       1. If _v1_ or _v2_ is *-0*<sub>𝔻</sub>, then
-        1. If _d1_ and _d2_ are both negative, return Decimal128ValueToObject(PickQuantum(*+0*<sub>𝔻</sub>, _newQ_)).
+        1. If sign(_d1_) = -1 and sign(_d2_) = -1, return Decimal128ValueToObject(PickQuantum(*+0*<sub>𝔻</sub>, _newQ_)).
         1. Return Decimal128ValueToObject(PickQuantum(*-0*<sub>𝔻</sub>, _newQ_)).
       1. Let _product_ be _v1_ × _v2_.
-      1. Assert: _product_ ≠ 0.
-      1. Let _roundedProduct_ be RoundToDecimal128Domain(_product_).
-      1. Return Decimal128ValueToObject(PickQuantum(_roundedProduct_, _newQ_)).
+      1. Return Decimal128ValueToObject(PickQuantum(_product_, _newQ_)).
```

Comment:
Given that PickQuantum cannot accept a 0, can we leave the assertion?

Reply:
Sure, I've added it back in.